library/data/registry_entry_attachments/src/main/java/org/quiltmc/qsl/registry/attachment/impl/RegistryEntryAttachmentHolder.java

```diff
@@ -134,7 +129,12 @@ public <T> void putValue(RegistryEntryAttachment<R, T> attachment, TagKey<R> tag
 		this.valueTagTable.put(attachment, tag, value);
 	}
 
+	protected <T> void putValueFromTag(RegistryEntryAttachment<R, T> attachment, R entry, T value) {
+		putValue(attachment, entry, value);
+	}
+
 	public void clear() {
 		this.valueTable.clear();
+		this.valueTagTable.clear();
```

Comment:
oops

Reply:
Hey, at least we noticed it before any end users :P